cmd/kfctl/cmd/replicate-build.go

```diff
@@ -0,0 +1,59 @@
+package cmd
+
+import (
+	"fmt"
+	kftypes "github.com/kubeflow/kfctl/v3/pkg/apis/apps"
+	"github.com/kubeflow/kfctl/v3/pkg/kfapp/gcp"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var include string
+var exclude string
+var context string
+var outputFileName string
+
+func init() {
+	replicateBuildCmd.Flags().StringVarP(&include, "include", "i", "", "When set, only replicate images with matching prefix")
```

Comment:
Instead of using flags to specify include/exclude patterns, can we define a suitable YAML file which lists the patterns and replacements?